worker/src/index.ts

```diff
@@ -58,8 +58,12 @@ export default {
     state.overallDown = 0
     state.overallUp = 0
 
+    let statusChaged = false
```

Comment:
You made a typo here, it should be `statusChanged`. So your reference to `statusChanged` below actually refers to an undefined variable.